wireguard-install.sh

```diff
@@ -32,195 +243,73 @@ function addClient() {
 
 	# Generate key pair for the client
 	CLIENT_PRIV_KEY=$(wg genkey)
-	CLIENT_PUB_KEY=$(echo "$CLIENT_PRIV_KEY" | wg pubkey)
+	CLIENT_PUB_KEY=$(echo "${CLIENT_PRIV_KEY}" | wg pubkey)
 	CLIENT_PRE_SHARED_KEY=$(wg genpsk)
 
+	# Home directory of the user, where the client configuration will be written
+	if [ -e "/home/$CLIENT" ]; then # if $1 is a user name
+		HOME_DIR="/home/$CLIENT"
+	elif [ "${SUDO_USER}" ]; then # if not, use SUDO_USER
+		HOME_DIR="/home/${SUDO_USER}"
+	else # if not SUDO_USER, use /root
+		HOME_DIR="/root"
+	fi
```

Comment:
This doesn't work, as root my config file is written to `/home//wg0-client-xxxx.conf`. Could we keep using the native `$HOME` variable? Works fine with both root and sudo

Reply:
The code works and fixes #96 , we do the same for the openvpn-install script. The problem is that I forgot to rename `$CLIENT` from the openvpn-install repository to `$CLIENT_NAME` which is the name we use in this repository. Thanks for noticing :smiley:

Regarding L250-L251, since I didn't want to make a bigger PR the code to generate custom client names instead of random ones is found in 7426347bcd4829aed0ad25fefd9bcf05ae7c43c9 of #94